docs/8.Outputs/CycloneDX BOM.md

```diff
@@ -206,3 +262,42 @@ ex.
 ```shell
 CHECKOV_CYCLONEDX_SCHEMA_VERSION=1.3 checkov -d . -o cyclonedx
 ```
+
+## Structure
+
+Further information on the different elements and attributes can be found [here](https://cyclonedx.org/docs/1.4/xml/).
+
+### component
+
+Each component stores the information of a single IaC resource or SCA package.
+
+IaC
+```xml
+<component bom-ref="pkg:terraform/main.tf/aws_s3_bucket.example@sha1:c9b9b2eba0a7d4ccb66096df77e1a6715ea1ae85" type="application">
+  <name>aws_s3_bucket.example</name>
+  <version>sha1:c9b9b2eba0a7d4ccb66096df77e1a6715ea1ae85</version>
+  <hashes>
+    <hash alg="SHA-1">c9b9b2eba0a7d4ccb66096df77e1a6715ea1ae85</hash>
+  </hashes>
+  <purl>pkg:terraform/main.tf/aws_s3_bucket.example@sha1:c9b9b2eba0a7d4ccb66096df77e1a6715ea1ae85</purl>
+</component>
+```
+
+- `name`: Resource ID
+- `version`: sha1 hash of the file
+- `hash`: sha1 hash of the file
+- `purl`: Format `pkg:<runner name>/<file path>/<resource ID>@<sha1 hash of the file>`
+
+SCA
+```xml
+<component bom-ref="pkg:pypi/flask@0.6" type="library">
+  <name>flask</name>
+  <version>0.6</version>
+  <purl>pkg:pypi/flask@0.6</purl>
+</component>
+```
+
+- `group`: Group name of the package
+- `name`: Name of the package
+- `version`: Version of the package
+- `purl`: Format `pkg:<package type>/<package group name>/<packge name>@<package version>`
```

Comment:
Awesome, informative, and clear.

Reply:
Do we want to add the repo name or ID?
for a case that we are merging two repos reports.